Lang/Spanish.pj.Lang

```diff
@@ -419,7 +420,7 @@
 //Edit Cheat
 #1027#  "Editar Truco"
 #1028#  "Actualizar Truco"
-#1029#  "El Truco ha sido cambiado.\n\n¿Deseas actualizar?"
+#1029#  "El Truco ha sido cambiado.\n\n¿Quieres actualizar?"

```

Comment:
Understood.

Please take a look at this book fragment:
https://books.google.com/books?id=AGJwGzTJaS4C&pg=PA40&lpg=PA40&dq=deseas+o+quieres&source=bl&ots=1sTsZDBFb4&sig=CW_plCgutBmwOBw_GYhliJQR_So&hl=en&sa=X&ved=0ahUKEwjcvpTY1rrJAhWElIgKHYalDC0Q6AEIJjAF#v=onepage&q=deseas%20o%20quieres&f=false

Note that both "Quieres" and "deseas" are valid to express an offering.
However, "deseas" is used more often in formal/professional contexts (i.e. in a restaurant, when they ask whether you wish to order).